### ISSUE TITLE
this step not clear

### ISSUE THREAD

**Issue by awnosa** [rainforestapp/tester-chrome-extension#687]
what assessment do you want and not under stand this step

![fsfs](https://cloud.githubusercontent.com/assets/16338438/19203174/aedf2ffc-8c8a-11e6-88d4-9069361ef58a.jpg)

**Comment by Electroglitch** [comment]
That Test author in general needs some help from Rainforest customer success team of what ever the name may be and explain the process of writting test steps and help him write CLEAR and SIMPLE test
written in mind that not everyone understands particular user interface and some terms as familiar.

**Comment by sweeneyapps** [comment]
I do understand this but yeah others may not understand.   

The tester author is asking you to click one of the active assessments's name such like Math Sample Full v1 to see if your access is limited.   They are checking to see if you can't launch any of the active assessment.  

but yeah, it could be explained clearer.    If you don't understand, you can click No, and explain why you don't understand this.    That will help Test Author to adjust instruction and question in the future.

**Comment by HW71** [comment]
@awnosa 
Just my 2 cents:
If something is not clear with the instruction then you should fail the test / answer "No" and simply describe what's wrong with the test specification / instruction in a polite way in the "Looks like you found a bug" modal. This information will go to the customers, so you should not be rude or pretentious.
If I got the impression that the test author is not very experienced with writing test specs I sometimes add a hint on how to clarify things (but actually I'm not quite sure if that's ok so please don't take that for granted).

In any case you should also add a summary about what was wrong in a more catchy way to the Notes field in the survey at the end of the test because this information will go to some RF folks which are in contact with the customers, so they also get some information what's going on and maybe get a better feeling on how to support them in getting "better" test authors... So to say... :-)

**Comment by Electroglitch** [comment]
@sweeneyapps It is not the point if you and i understand ,it has to be written clearly so everyne can understand. Just like Rainforest is training testers,
 you don't have to think what to do ,you just follow the instructions and it has to be crystal clear and in this case it is not.
Why is it so hard to write it like you explained fellow tester above,
 i'm repeating myself but not everyone is familiar with some terms ,user interface and interaction.

**Comment by qpPykT** [comment]
@sweeneyapps 
'An' or 'any' assessments's name. Can it be used in the same way?

**Comment by mobradovic** [comment]
> 'An' or 'any' assessments's name. Can it be used in the same way?

Pick an apple from the pile.
Pick any apple from the pile.

Same thing.

**Comment by sweeneyapps** [comment]
@qpPykT  yeah just like @mobradovic pointed it out.   👍

**Comment by sweeneyapps** [comment]
@Electroglitch 

in some case..  the instruction is simple clear and the Tester Author's exceptions is higher on us.   

For example, "please download this file then click browse to select the files".      It's clearly the Test Author expected tester to have the knowledge of downloading files and finding the files themselves.    We do have the skill to understand and how to do this.   I have seen a tester saying No, and writing an explanation that says, "You should add more steps such like opening a new tab, insert the link in new tab, then open the explorer to see if file is downloaded, then blah blah.. goes on.. more steps for this instructions when the Test Author expected tester to have the skill to download file themselves.   To be honest, downloading files is basic computer skill that all testers should have.

Another case where the Test Author is only trying to test to see if UI is friendly enough for normal users to navigate through, so they use testers to try out the UI and see if it actually clear enough.

Clear or not, in this assignment, it's clear to me because it's only asking me to pick ONE assignment name and on the UI, there's list of active assessments.   It seems so simple.    But I'm saying.. for others.. it's OK to click NO and explain why.

It's possible that previous steps are connected.. and the Test Author expected you to fully understand what you are doing.   Like starting from step 1, you clicked on assessments and you changed the assessment name.. etc.   Clearly, you know it's list of assessments below.   You are learning from the beginning to the end.

**Comment by HW71** [comment]
What should be considered here, is, that writing good test specs is also a task that needs to be learned! Especially (but not only) if "English" is not the native language of the test author.

So yes, @Electroglitch is absolutely right: there are tests ongoing where TAs need some additional support from RF, because our task is definitely not to "guess". If something is not clear for whatever reason then we are asked to fail the test, which means less money on our side - and a "red" test suite on customer side. So by the end of the day a clear lose-lose situation for all of us.

Of course TAs could also expect "basic skills", but who defines that? And what about different OSs? In any case this is definitely an interesting discussion and should be followed-up (but maybe this issue tracker here is not the right place to do so... :-)   )...

**Comment by sweeneyapps** [comment]
yeah, can't wait for the forum to open to everyone. :)

**Comment by clessweapon** [comment]
I usually report on the first step, the test asks if see "random name" and "Dasboard"? But there are two "Dashboard" (a button and a text bleow)the instruction does not specify what should I consider.
I am wrong?

**Comment by nihlathar** [comment]
@clessweapon if the step told you to click on dashboard, in this case, then you couldn't complete the step  because you don't know which one to click, but if it asks if you see it at all, you say yes because you do (multiple)

**Comment by Electroglitch** [comment]
lol Google or google translate for some users as we all know,you really think that's the way test should be done?

If i was a test author and paying big money i would make sure that my test are easy to understand and follow, not to guess, point of Rainforest is that average user can do the test ,remember the line , do every test like you are doing it for the first time or something like that :)

I ve' been doing this for more than 2 years and there was the time before strict rules where the tester was filling the gap between incomplete step instructions and actual situation on page but that time is over...
Do i have to remind you what's written on Wiki page :
"Being a great tester is purely about following the instructions in the test to the letter, and passing or failing the test as the steps require. Nothing else."

Speaking about test author above i really don't see why is it problem to write something-like -Do you se username "some name" in top right corner and Title "some title" on the top left side
Click on the "some button" in top nav bar etc 
Using screenshot example to point out something that average internet user could find unfamiliar.
You all have done RF application test and know how easy it is to put screenshot in step ,just drag & drop pic in the field...

I could rewrite that the whole test in couple of minutes so there would be no confusion what do do,just follow the instructions.

I mean c'mon some test authors can write test containing 80-100 steps and doing it feels so smooth,
on the other hand there are some authors struggling to put 3 or 4 steps toghether.

And yes i blame Rainforest customer success team the most, it's their job not mine,
how many test, minutes and hours  i've spent writting and explaining test author proper usage of placeholder for example... c'mon!

**Comment by RnkSts** [comment]
Click on **an** (_Task author no specify which one, so he don't care which one_) assessment name (_Look around for an assessment, yes (: , there is a list_) and **verify** the **Proctor** (_The user, you_) can**not** start taking the assessment exam (_Something prevent you to continue_) which **would appear** in the next window.

Do you see that you can **NOT** launch the Assessment? (_Any message or something that says you 'no, get out of here' xd_).

One step is enough for this, click an assessment, you should get a message that can't continue or something (at least there's a bug) and click YES. It's not too complicated, you maybe need to read it again and ask yourself what is the intention of the author.

Some people may not understand what to do and that's Okay, We don't have any previous experience in the sites we test, but we can learn from the other test we do before. And Task authors also need to know this and make the instructions clear to follow.

We are here for testing websites, so task author expect that we have enough knowledge in what we are doing, I see some cases like in bluecore tests, in the Sign In whit Google step, some report as NO because when you enter the email you need to click "next" to enter the password and click another button to Sign In, and the instruction only say something like  'Click Sign In with Google and use this credentials din dan don...' and it no says step-by-step how to do that.

Same with uploading documents, or one I remember one that says something like "Download a photo from the web and upload to the app", so you need to open a new tab, search a random thing on Google images, Pick an image, Save the image, Click the upload button an look for the file in the File Explorer and upload it. All in a single step, a thing that maybe we normally do.

We also have this entry in the confusing scenarios section on the wiki:
>- If the instructions are intentionally vague and general, follow them as closely as you can. You should report any errors that show up and anything that prevents you from completing the task.

Some people get upset about training jobs (_that's okay if you receive 50 in a day xd_), but they are here for one reason, improve our experience.

@Electroglitch Using Google may be useful, if you see a technical word, you can google it and know what it means, or if the job ask to select for example "Scatterplot" option and make sure the app gives you the correct graph, you can google it an verify, if you have not see it before.

**Comment by nerdatwork** [comment]
This also shows testers who do not properly understand English would find it very hard to follow the instructions same as OP.

@awnosa If you are satisfied with the responses you got so far, I suggest you to close this thread.

**Comment by clessweapon** [comment]
> 
Same with uploading documents, or one I remember one that says something like "Download a photo from the web and upload to the app", so you need to open a new tab, search a random thing on Google images, Pick an image, Save the image, Click the upload button an look for the file in the File Explorer and upload it. All in a single step, a thing that maybe we normally do.

I usually report this type of hit because of the rule "Do not perform any actions que are not specified." Unfortunately, the Wiki does not mention if open new tabs without instruction is allowed or not, or if kristen has given an official answer to this question.

**Comment by RnkSts** [comment]
@clessweapon from my side I think the action is specified, "_Download a photo from the web_", We have complete some test like this, except they give us a link (Download host.some/file.csv an upload it). But you're right, the wiki need more content and the forum will bring it to all that care about this.

**Comment by awnosa** [comment]
thanks to all for help me
